README.md

```diff
@@ -0,0 +1,49 @@
+## Merging selinux-policy-contrib repository with selinux-policy
+
+Currently, SELinux policy packages in Fedora use 2 repositories:
+base [1] and contrib [2].
+This division into two repos is merely a historical artifact and
+the main effect is make dealing with the repos more difficult.
+
+We work on a change to merge these repos into one, containing sources from both.
+All the changes would affect both the repos, branches rawhide, and future branches
+f34 and newer.
```

Comment:
I would suggest to mark up the branch names and change word order a bit:
```
All the changes would affect both repos, `rawhide` branches, and future branches
`f34` and newer.
```

Reply:
Updated.